addons/mail/static/src/components/rtc_option_list/rtc_option_list.xml

```diff
@@ -8,10 +8,18 @@
                     <i class="o_RtcOptionList_button_icon fa fa-lg fa-th-large"/>
                     <span class="o_RtcOptionList_button_text">Change layout</span>
                 </button>
-                <button class="o_RtcOptionList_button" t-on-click="rtcOptionList.onClickFullScreen">
-                    <i class="o_RtcOptionList_button_icon fa fa-lg fa-arrows-alt"/>
-                    <span class="o_RtcOptionList_button_text">Full screen</span>
-                </button>
+                <t t-if="rtcOptionList.rtcController.callViewer.isFullScreen">
```

Comment:
It seems like a long chain of relations without ever verifying that they are all set.
Also those respective relation fields are not currently marked as required, so it makes it even less safe to trust.

As there is a condition at least on "rtcOptionList" at the top, this is already something but I would be more comfortable if all other fields that you followed were marked as required (and potentially also isCausal and readonly), or if they can actually be undefined then a condition should be added for them.